src/utils/helper.ts

```diff
@@ -96,3 +114,14 @@ export function deleteRoleApprovalIfExist(roleApprovalId: string): void {
     store.remove('RoleApproval', roleApprovalId)
   }
 }
+
+export function findOrCreateRolesRegistry(rolesRegistryAddress: string): RolesRegistry {
+  let rolesRegistry = RolesRegistry.load(rolesRegistryAddress)
```

Comment:
We have been using lower case address when used in an ID. We can change that, but we should change it everywhere to make it consistent (I would review other generate id functions as well)